build.gradle

```diff
@@ -60,24 +64,24 @@ sourceSets {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:4.1.0')
+    api platform('com.exactpro.th2:bom:4.6.0')
```

Comment:
note: if you choose to migrate to th2 gradle plugins this won't be needed here anymore